### PR TITLE
Glass shards can now be crafted by using glass in-hand

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request

Activate a glass sheet in-hand to craft it into a glass shard. 

## Why It's Good For The Game

QoL. Quicker than crafting a window then smashing it, and it makes sense you can snap a glass sheet in half. 

## Changelog
:cl:
add: Glass shards can now be made by using glass in-hand. 
/:cl:

### CODE REVIEW

**Comment by utilbird** [comment]
what if it.. embedded shards in  your hands without gloves... just kidding.. unless..?

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
  * Glass sheets
  */
 GLOBAL_LIST_INIT(glass_recipes, list ( \
+	new/datum/stack_recipe("glass shard", /obj/item/shard, time = 0, on_floor = TRUE, window_checks = TRUE), \
```
Comment (PowerfulBacon) [inline code/game/objects/items/stacks/sheets/glass.dm]: Why are you doing window checks and putting it on floor for a glass shard

Comment (qwertyquerty) [reply code/game/objects/items/stacks/sheets/glass.dm]: this tbh

Comment (VictorVow) [reply code/game/objects/items/stacks/sheets/glass.dm]: Don't copy and paste kids

**Comment by utilbird** [comment]
This should require a tool to do, I'm not sure if you've tried to snap a glass sheet before, it's not easy

**Comment by utilbird** [comment]
image

**Comment by VictorVow** [comment]
Just stomp it, voila.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**